src/main/scala/guessbook/GuessBook.scala

```diff
@@ -0,0 +1,111 @@
+package guessbook
+
+import com.github.tototoshi.csv.CSVReader
+import guesswho.GenderEnum
+
+import scala.util.{Failure, Random, Success, Try}
+
+object GuessBook {
+
+  // Read book info from csv file
+  def getBooksFromCSV(path: String): Seq[Book] = {
```

Comment:
I've never done anything like this so can't say for certain, but it looks good to me, especially the for comp in here :D